.gitignore

```diff
@@ -1 +1,3 @@
 node_modules
+dist
+cache
```

Comment:
It's .cache and for don't check the dist, to must remove before and later add and commit